scripts/build_subtensor_linux.sh

```diff
@@ -0,0 +1,23 @@
+#!/bin/bash
+
+# Install dependencies
+sudo apt-get update && sudo apt-get install -y curl build-essential protobuf-compiler clang git
+
+# Install rust
+curl https://sh.rustup.rs -sSf | sh -s -- -y
+
+# Activate the rust environment and setup path
+source "$HOME/.cargo/env"
+
+# Get the subtensor repo
+git clone https://github.com/opentensor/subtensor.git
+cd subtensor
```

Comment:
in this case is the same but taking this as an opportunity to share with you a typical approach in bash scripts.
and that's using pushd and popd.

so in this case we would...
```suggestion
pushd subtensor
...
...
popd # instead of cd ..
```

Reply:
Interesting, nice to know!